Formula/hasura-cli.rb

```diff
@@ -14,22 +15,27 @@ class HasuraCli < Formula
   end
 
   depends_on "go" => :build
+  depends_on "node" => :build
 
   def install
+    Language::Node.setup_npm_environment
+
     ldflags = %W[
       -s -w
-      -X github.com/hasura/graphql-engine/cli/version.BuildVersion=#{version}
-      -X github.com/hasura/graphql-engine/cli/plugins.IndexBranchRef=master
-    ]
+      -X github.com/hasura/graphql-engine/cli/v2/version.BuildVersion=#{version}
+      -X github.com/hasura/graphql-engine/cli/v2/plugins.IndexBranchRef=master
+    ].join(" ")
 
     cd "cli" do
-      system "go", "build", *std_go_args, "-ldflags", ldflags.join(" "), "-o", bin/"hasura", "./cmd/hasura/"
-
-      system bin/"hasura", "completion", "bash", "--file", "completion_bash"
-      bash_completion.install "completion_bash" => "hasura"
+      ENV["CI"] = "false"
+      system "make", "build-cli-ext"
+      system "make", "copy-cli-ext"
```

Comment:
```suggestion
      with_env(CI: "false") do
        system "make", "build-cli-ext"
        system "make", "copy-cli-ext"
      end
```
Let's try to preserve the environment where we can.

Reply:
👍